### DISCUSSION TITLE
Predicting for a saved TimeseriesForecaster model

### DISCUSSION THREAD

**Discussion by pingusix** [keras-team/autokeras#1626] [Q&A]
Hi

I'm currently playing with the TimeseriesForecaster and have a question about running a prediction on a saved model.

I can fit and predict well enough but can't figure out how to use a saved model for a prediction.

If I run an export_model after a fit and then use model.save I get that the exported model has input shape (None, 3, 12) where my lookback is 3 and there are 12 input columns.

When I load my saved model with load_model("saved_model_name", custom_objects=ak.CUSTOM_OBJECTS), I get that the input_shape is the same as the exported model ie: (None,3,12) as expected.

What I can't figure out is how to get my test data 12 column dataframe into the correct format for the predict function to work.

Any help would be appreciated.

Cheers
Ed

**Comment by Chiarinni** [comment]
I have the same problem
